stripe-patrons-data/README.md

```diff
@@ -1,6 +1,22 @@
-## stripe-patrons-data
+This stack consists of 3 Lambdas.
 
-This project is a simple, scheduled lambda function which will query Stripe for data on active Guardian Patrons and insert them into the SupporterProductData DynamoDB table
+The lambdas are [versioned](https://docs.aws.amazon.com/lambda/latest/dg/configuration-versions.html) so that they can use [Snapstart](https://aws.amazon.com/about-aws/whats-new/2022/11/aws-lambda-snapstart-java-functions/) for faster start-ups.
+
+This means that the lambdas are initialised during the riffraff deploy, resulting in faster invocations.
+
+Config is fetched from SSM during initialisation, not during invocation. This means that if you want to change the lambda config you need to also deploy a different build number - otherwise a new lambda version will not be created and the new config will not be picked up.
```

Comment:
I think this could be achieved by adding the `workflow_dispatch:` trigger ([example](https://github.com/guardian/slo-alerts/blob/3d11e60c359982fa1523f4e59f5d26238467a663/.github/workflows/ci.yaml#L2-L7)) to your GitHub Actions config. This would allow developers to easily [run a new build for `main` via the GH Actions UI](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) and I presume this would give you an identical build with a different `GITHUB_RUN_NUMBER` (and therefore the opportunity to reload config!).

Obviously there are other ways to do this quickly (e.g. editing the `README` and deploying a branch!) but as GH Actions is relatively new it might be worth documenting the recommended approach here?